coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 21.72
+    "covered_percent": 83.44
```

Comment:
I promise this is what it looked like on my local machine. Idk why it was getting pushed as 21.72.